addons/hr_expense/models/account_move_line.py

```diff
@@ -8,7 +8,7 @@
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
-    expense_id = fields.Many2one('hr.expense', string='Expense', copy=False)
+    expense_id = fields.Many2one('hr.expense', string='Expense', copy=True)
```

Comment:
Another solution would be to make `expense_id` computed based on `reversed_entry_id`, this way, we still don't copy outside of reversal

Reply:
well, I thought about keeping the same behavior for when it's not a reversal... but I think even when you duplicate manually a bill created from an expense, you would not understand why the computation isn't the same.

Don't you agree? :)

Reply:
You're right